src/main/scala/com/example/Service/ReservationService.scala

```diff
@@ -0,0 +1,148 @@
+package com.example.Service
+
+import java.sql.Timestamp
+
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.{ActorRef, Behavior}
+import akka.pattern.StatusReply
+
+import scala.util.{Failure, Success}
+import com.example.Registry.Registry.{CreateReservationDB, GetReservationsScreeningDB, GetScreeningScreeningRoomDB, GetScreeningTimeDB}
+import com.example.Registry.{Registry, ReservationCreatedDB, ReservationsScreeningDB, ScreeningScreeningRoomDB, ScreeningTime}
+
+import scala.concurrent.duration._
+import akka.util.Timeout
+import pureconfig.ConfigSource
+
+import scala.util.matching.Regex
+import pureconfig.generic.auto._
+
+final case class Reservation(screeningId: Int, name: String, seats: Seq[(Int, Int)], adult_tickets: Int, student_tickets: Int, child_tickets: Int)
+final case class ReservationResponse(total_amount: String, expiration_time: Timestamp)
+
+final case class Prices(adultTicketsPrice: Double, studentTicketsPrice: Double, childTicketsPrice: Double)
```

Comment:
do you think Double is right type here?

Reply:
i guess there is no need for such precision, it should be float type

Reply:
What about BigDecimal? Remeber, that we are talking about money here

Reply:
you're right, there s no room for mistakes